Examples/DicomSeriesFromArray/Documentation.rst

```diff
@@ -0,0 +1,26 @@
+.. _lbl_dicom_series_from_array:
+
+Dicom Series From Array
+==============================
+
+Overview
+--------
+This example illustrates how to write a DICOM series from an array in Python
```

Comment:
Please remove the reference to Python, SimpleITK supports 8 languages so we keep the documentation language neutral as it is shared across languages.